### ISSUE TITLE
dashboard mode setting not reflected in shell

### ISSUE THREAD

**Issue by gureckis** [NYUCCL/psiTurk#514]
for someone launching shell and dashboard at same time the mixed states is confusing.  you can have the dashboard talking to 'live' and the shell still talking to 'sandbox'

**Comment by deargle** [comment]
I had to be really careful with the dashboard mode, since multi server
threads could lead to crazy conditions of the mode changing on page
refresh. The dashboard sets and reads a server cookie to know the mode. But
are you saying the mode used at config launch is not used by the dashboard?
I think it's impossible for the shell to know the browser's mode cookie,
and vice versa, impossible for the dashboard to know any shell's current
mode.

**Comment by deargle** [comment]
Yeah short of storing the current mode in a database "settings" table somewhere (overkill, I think), I think there's no way around this. Do you want to update the docs somewhere to point this out? Where would be good?
